mscore/prefsdialog.cpp

```diff
@@ -1235,9 +1235,9 @@ void PreferenceDialog::applyPageVertical()
                   ss->doLayout();
             }
       if (cv)
-            cv->setOffset(0.0, 0.0);
+            cv->pageTop();
       if (mscore->currentScoreView())
-            mscore->currentScoreView()->setOffset(0.0, 0.0);
+            mscore->currentScoreView()->pageTop();
```

Comment:
Do line 1239 and 1240 make sense, or can they be deleted? They seem redundant to me, since `cv` _is_ `mscore->currentScoreView()` (see some lines above).

Reply:
mm, it's quite strange actually, looks like a duplication
let's remove it

Reply:
I did that, and it doesn't seem to cause any problems. So let's indeed remove it!